pluralforms.json

```diff
@@ -0,0 +1,282 @@
+{
```

Comment:
Where this information comes from? Did you write your own with reference to CLDR?

Reply:
This is from https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html
which ultimately comes from cldr, yes